sdk/test/metrics/BUILD

```
@@ -9,3 +9,27 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "view_registry_test",
+    srcs = [
+        "view_registry_test.cc",
+    ],
```

Would it be possible to add tags here?
```suggestion
    ],
    tags = [
        "metrics",
        "test",
    ],
```

---

Thanks. Added it.